### PR TITLE
remove ui from go tests

### PR DESCRIPTION
**Pull request by johnnyaug**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -168,8 +168,12 @@ esti: ## run esti (system testing)
 
 test: test-go test-hadoopfs  ## Run tests for the project
 
-test-go: gen			# Run parallelism > num_cores: most of our slow tests are *not* CPU-bound.
-	$(GOTEST) -count=1 -coverprofile=cover.out -race -cover -failfast --parallel="$(GOTEST_PARALLELISM)" $(GO_TEST_MODULES)
+make-ui-dir:
+	mkdir ./webui/dist
+	touch ./webui/dist/index.html
```
Comment (nopcoder) [inline Makefile:173]: Not sure we would like it in the Makefile - or alternative enable specific step that will just generate the Go files that will skip the node/webui stuff.

File: clients/spark/build.sbt [hunk 2]
```diff
@@ -123,23 +126,30 @@ lazy val spark3Type =
...
-lazy val core312 = generateCoreProject(spark312Type)
-lazy val examples2 = generateExamplesProject(spark2Type).dependsOn(core2)
-lazy val examples3 = generateExamplesProject(spark3Type).dependsOn(core3)
-lazy val examples312 = generateExamplesProject(spark312Type).dependsOn(core312)
+// lazy val core2 = generateCoreProject(spark2Type)
+// lazy val core3 = generateCoreProject(spark3Type)
+lazy val root = generateCoreProject(spark312Type)
+// lazy val examples2 = generateExamplesProject(spark2Type).dependsOn(core2)
+// lazy val examples3 = generateExamplesProject(spark3Type).dependsOn(core3)
+// lazy val examples312 = generateExamplesProject(spark312Type).dependsOn(core312)
```
Comment (nopcoder) [inline clients/spark/build.sbt:143]: Part of the PR?

**Review by nopcoder — CHANGES_REQUESTED** [review]

File: clients/spark/build.sbt (hunk 2 above)
Comment (johnnyaug) [reply clients/spark/build.sbt:143]: oops!

**Review by nopcoder — APPROVED** [review]
LGTM

**Pull request merged**